Project generated with archetype fails to compile

```
[INFO] Scanning for projects...
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building A Hazelcast Simulator Test project 1.0-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ yourproject ---
[INFO] 
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ yourproject ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory /home/peter/yourproject/src/main/resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.1:compile (default-compile) @ yourproject ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 1 source file to /home/peter/yourproject/target/classes
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 0.681 s
[INFO] Finished at: 2015-06-07T11:01:36+03:00
[INFO] Final Memory: 26M/1963M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:compile (default-compile) on project yourproject: Fatal error compiling: invalid target release: 1.8.0_45 -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

--- a/archetype/src/main/resources/archetype-resources/pom.xml
+++ b/archetype/src/main/resources/archetype-resources/pom.xml
@@ -13,7 +13,7 @@
     <properties>
         <hazelcast.version>${hazelcast.version}</hazelcast.version>
         <hazelcast-simulator.version>${project.version}</hazelcast-simulator.version>
-        <java.version>${java.version}</java.version>
+        <java.version>1.6</java.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     </properties>
 
@@ -74,8 +74,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.1</version>
                 <configuration>
-                    <source>${java.version}</source>
-                    <target>${java.version}</target>
+                    <source>\${java.version}</source>
+                    <target>\${java.version}</target>
                 </configuration>
             </plugin>
             <plugin>

